mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -125,17 +118,18 @@ impl CommandHandler {
                 }
 
                 if subacks.is_empty() {
-                    return Ok(true);
+                    return Ok(());
                 }
             }
         }
 
-        Ok(false)
+        Err(CommandHandlerError::MissingSubacks(topics.join(", ")))
     }
 
     async fn handle_event(&mut self, event: Event) -> Result<(), HandleDisconnectError> {
         if let Event::Publication(publication) = event {
-            let client_id = parse_client_id(&publication.topic_name)?;
+            let client_id = str::from_utf8(&publication.payload)
```

Comment:
I think we should parse client_id (or a list of ids) from a json

Reply:
I went with single id